src/lib/server/prisma.ts

```diff
@@ -1,5 +1,4 @@
 import { PrismaClient } from "@prisma/client";
 
 const prisma = new PrismaClient();
-await prisma.$connect();
```

Comment:
Removes the need for the env variable to be defined at build-time (even if it just needs to be a dummy value)

Reply:
I'll see if I can separate this into something we can upstream :eyes:

Reply:
Done!